main.go

```diff
@@ -18,6 +18,7 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"go.uber.org/automaxprocs/maxprocs"
 	"os"
 
 	"github.com/ianschenck/envflag"
```

Comment:
```suggestion
	"os"

	"github.com/ianschenck/envflag"
	"go.uber.org/automaxprocs/maxprocs"
```
imports should be grouped according to https://go.dev/wiki/CodeReviewComments#imports